src/PointwiseFunctions/AnalyticData/GrMhd/BondiHoyleAccretion.cpp

```diff
@@ -170,6 +170,17 @@ BondiHoyleAccretion::variables(
       x);
 }
 
+template <typename DataType>
+tuples::TaggedTuple<hydro::Tags::ElectronFraction<DataType>>
+BondiHoyleAccretion::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<hydro::Tags::ElectronFraction<DataType>> /*meta*/) const
+    noexcept {
+  return {
+      make_with_value<db::item_type<hydro::Tags::ElectronFraction<DataType>>>(
+          x, 0.1)};
```

Comment:
The different solutions seem to choose arbitrarily between this and `make_with_value<Scalar<DataType>>`.  Can they be made consistent?

Reply:
Yes, I think we need to scour the code for `db::item_type` and get rid of it when the type is known, but to do so in the PR either requires me to do that for various files now, or make the functions inconsistent within a given file.  So would you prefer:
- A commit in this PR updating the currently touched files
- Fixing this only in the lines I touched in this PR
- An issue filed to purge `db::item_type` everywhere it can be done and change nothing
- A PR that purges `db::item_type` and this PR then depends upon it

Reply:
Ah, I didn't realize it was following the existing style of each file.  I'm fine with this PR as-is then.

You can file an issue if you wish.  I don't know the argument for using one form over the other, so I'm not going to specifically ask for one.